src/components/indicators/indicators.js

```diff
@@ -129,43 +109,37 @@ define(['datetime', 'itemHelper', 'emby-progressbar', 'css!./indicators.css', 'm
 
         if (item.SeriesTimerId) {
             if (status !== 'Cancelled') {
-                return '<i class="material-icons timerIndicator indicatorIcon fiber_smart_record"></i>';
+                return '<span class="material-icons timerIndicator indicatorIcon fiber_smart_record"></span>';
             }
 
-            return '<i class="material-icons timerIndicator timerIndicator-inactive indicatorIcon fiber_smart_record"></i>';
+            return '<span class="material-icons timerIndicator timerIndicator-inactive indicatorIcon fiber_smart_record"></span>';
         }
 
-        return '<i class="material-icons timerIndicator indicatorIcon fiber_manual_record"></i>';
+        return '<span class="material-icons timerIndicator indicatorIcon fiber_manual_record"></span>';
     }
 
-    function getSyncIndicator(item) {
+    export function getSyncIndicator(item) {
         if (item.SyncPercent === 100) {
-            return '<div class="syncIndicator indicator fullSyncIndicator"><i class="material-icons indicatorIcon file_download"></i></div>';
+            return '<div class="syncIndicator indicator fullSyncIndicator"><span class="material-icons indicatorIcon file_download"></span></div>';
         } else if (item.SyncPercent != null) {
-            return '<div class="syncIndicator indicator emptySyncIndicator"><i class="material-icons indicatorIcon file_download"></i></div>';
+            return '<div class="syncIndicator indicator emptySyncIndicator"><span class="material-icons indicatorIcon file_download"></span></div>';
         }
 
         return '';
     }
 
-    function getTypeIndicator(item) {
-        if (item.Type === 'Video') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon">videocam</i></div>';
-        }
-        if (item.Type === 'Folder') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon">folder</i></div>';
-        }
-        if (item.Type === 'PhotoAlbum') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon photo_album"></i></div>';
-        }
-        if (item.Type === 'Photo') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon">photo</i></div>';
+    export function getTypeIndicator(item) {
+        const iconT = {
+            'Video' : 'videocam',
+            'Folder' : 'folder',
+            'PhotoAlbum' : 'photo_album',
+            'Photo' : 'photo'
         }
 
-        return '';
+        return iconT[item.iconType] ? '<div class="indicator videoIndicator"><span class="material-icons indicatorIcon '+ iconT[item.iconType] +'"></span></div>' : '';
```

Comment:
```suggestion
        const icon = iconT[item.iconType];
        return icon ? `<div class="indicator videoIndicator"><span class="material-icons indicatorIcon ${icon}"></span></div>` : '';
```

Reply:
What difference it makes to assign another variable a value first and then use that

Reply:
`iconT[item.iconType]` is referenced twice

Reply:
> `iconT[item.iconType]` is referenced twice

is that something bad :thinking:

Reply:
This is a question of performance. I doubt that JS caches value `iconT[item.iconType]` seeing it twice. At least it looks better to me - less duplication.

Reply:
I checked this by logging 10,000 direct references and another 10,000 using a local variable each time. The direct reference averaged 8,643 milliseconds over three runs while the local variable averaged 8,696 milliseconds. I wouldn't consider this a performance issue.

Reply:
<details>
<summary>My test</summary>

```html
<html>

<head>
<title>Reference test</title>

<script>
function getHtmlDirect(item) {
    const values = {
        '1111': 'AAAA',
        '2222': 'BBBB',
        '3333': 'CCCC',
        '4444': 'DDDD'
    };

    return values[item.type] ? '<span>' + values[item.type] + '</span>' : '';
}

function getHtmlCached(item) {
    const values = {
        '1111': 'AAAA',
        '2222': 'BBBB',
        '3333': 'CCCC',
        '4444': 'DDDD'
    };
    
    const value = values[item.type];
    return value ? '<span>' + value + '</span>' : '';
}

function runTest(fn) {
    const items = [{type: '1111'}, {type: '2222'}, {type: '3333'}, {type: '4444'}, {}];

    for (let i = 0; i < 10000; i++) {
        const item = items[i % items.length];
        const html = fn(item);
    }
}

function test(button) {
    button.disabled = 'disabled';
    
    const N = 1000;

    for (let i = 0; i < N; i++) {
        runTest(getHtmlDirect);
    }

    for (let i = 0; i < N; i++) {
        runTest(getHtmlCached);
    }

    button.disabled = '';
}
</script>
</head>

<body>
<button onclick="test(this)">Run test</button>
</body>

</html>
```
</details>

Chrome 79
![test-chrome](https://user-images.githubusercontent.com/56478732/81473976-18683700-920b-11ea-8ac7-623eb58d8979.png)

Reply:
<details>
<summary>Test with extracting of dictionary object</summary>

```html
<html>

<head>
<title>Reference test</title>

<script>
function getHtmlDirect(item) {
    const values = {
        '1111': 'AAAA',
        '2222': 'BBBB',
        '3333': 'CCCC',
        '4444': 'DDDD'
    };

    return values[item.type] ? '<span>' + values[item.type] + '</span>' : '';
}

function getHtmlCached(item) {
    const values = {
        '1111': 'AAAA',
        '2222': 'BBBB',
        '3333': 'CCCC',
        '4444': 'DDDD'
    };
    
    const value = values[item.type];
    return value ? '<span>' + value + '</span>' : '';
}

const values2 = {
    '1111': 'AAAA',
    '2222': 'BBBB',
    '3333': 'CCCC',
    '4444': 'DDDD'
};

function getHtmlDirect2(item) {
    return values2[item.type] ? '<span>' + values2[item.type] + '</span>' : '';
}

function getHtmlCached2(item) {
    const value = values2[item.type];
    return value ? '<span>' + value + '</span>' : '';
}

function runTest(fn) {
    const items = [{type: '1111'}, {type: '2222'}, {type: '3333'}, {type: '4444'}, {}];

    let arr = [];

    for (let i = 0; i < 10000; i++) {
        const item = items[i % items.length];
        const html = fn(item);
        arr.push(html);
    }

    console.debug(arr);
}

function test(button) {
    button.disabled = 'disabled';
    
    const N = 100;

    for (let i = 0; i < N; i++) {
        runTest(getHtmlDirect);
    }

    for (let i = 0; i < N; i++) {
        runTest(getHtmlCached);
    }

    for (let i = 0; i < N; i++) {
        runTest(getHtmlDirect2);
    }

    for (let i = 0; i < N; i++) {
        runTest(getHtmlCached2);
    }

    button.disabled = '';
}
</script>
</head>

<body>
<button onclick="test(this)">Run test</button>
</body>

</html>
```
</details>

Chrome 79
![test-chrome2](https://user-images.githubusercontent.com/56478732/81474486-67639b80-920e-11ea-9926-36705a8d3d1e.png)

Reply:
Firefox 72 shows that `Direct` wins :man_shrugging: 
But extracting a dictionary object makes a profit.

Reply:
> This is a question of performance. I doubt that JS caches value `iconT[item.iconType]` seeing it twice. At least it looks better to me - less duplication.

i agree it might affect performance in other cases but in this case we wont be seeing the performance affect at all since this is accessed in ternary only and its not like we are accessing it many times. According to my knowledge defining another `const` for that negates the gain we get for the second invocation of variable so performance might not matter. And about the tests you and @dkanada  are doing these are stress tests may not be valid in real world scenerio. 
As you said it looks better maybe from your point of view. But since im used to doing as i have done already so i did that way only. And Im totally aware this is an open source project so we must use some standard and think of others too. That might get better once i get familiar with proper standard of this project.
I was tasked to improve the code as suggested by @MrTimscampi so whatever i thought right i did :)